src/assets/src/components/queueSettings.tsx

```diff
@@ -135,6 +135,8 @@ export function ManageQueueSettingsPage(props: PageProps<SettingsPageParams>) {
     const [descriptValidationResult, validateAndSetDescriptResult, clearDescriptResult] = useStringValidation(queueDescriptSchema, true);
     const [allowedMeetingTypes, setAllowedMeetingTypes] = useState(new Set() as Set<string>);
     const [allowedValidationResult, validateAndSetAllowedResult, clearAllowedResult] = useMeetingTypesValidation(props.backends, queue);
+    const [queueLocation, setQueueLocation] = useState('')
+    const [locationValidationResult, validateAndSetLocationResult] = useStringValidation(queueNameSchema, true)
```

Comment:
For consistency with the other validation, can you also use `clearLocationResult` in this hook, and then add it to the `clearValidationResults` function? Without it, the validation message remains after saving, which isn't the worst, but is different behavior than the other fields.